Gemfile

```diff
@@ -1,19 +1,21 @@
 source 'https://rubygems.org'
 
+gem 'foreman', '~> 0.84'
```

Comment:
As with the Content Tagger change, I think the guidance on this from at least the guy who wrote foreman is to install foreman separately from the projects dependencies, and not have it in the Gemfile.

foreman doesn't seem to actually be used by router-api, only the new startup.sh script, and in the Dockerfile. The Gemfile in my mind contains gems needed to run (or develop) the application, of which foreman isn't one of them.